### DISCUSSION TITLE
在配置不同输出插件时，如何获取收集的日志中的指定字段值作为输出配置字段的值

### DISCUSSION THREAD

**Discussion by mumu-lab** [alibaba/ilogtail#956] [Help] [answered]
例如收集到的日志内容为：
{
  "__tag__:__path__": "/var/log/test.log",
  "platform_name": "test",
  "app_name": "demo",
  "env": "pro",
  "ip": "0.0.0.0",
  "port": "30113",
  "log_time": "2023-06-29 09:27:21.404",
  "log_level": "INFO",
  "pid": "7",
  "trace_id": "",
  "thread": "demo-1-C-1",
  "logger": "com.foundbyte.bigdata.infrastructure.utils.HBaseUtils",
  "log_info": "This is a test",
  "__time__": "1688002041"
}

如何配日志才能在配置输出时获取字段platform_name的值：
  如果输出到文件FileName的值如何设置为获取日志中字段platform_name的值
  如果输出到elasticsearch时index的值如何设置为获取日志中字段platform_name的值
 ...

**Comment by messixukejia** [comment, marked as answer]
这两个功能目前没有实现。是否可以一起来建设下？

如果可以参与共建，可以建个issue，并且assign给自己
